sycl/unittests/program_manager/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 set(CMAKE_CXX_EXTENSIONS OFF)
 add_sycl_unittest(ProgramManagerTests OBJECT
+  CompileTarget.cpp
```

Comment:
Shouldn't this be alphabetically sorted? Not sure really, some of the elements don't seem to be.